crates/pallet-executor/src/lib.rs

```diff
@@ -79,9 +81,10 @@ mod pallet {
             ensure_none(origin)?;
 
             log::debug!(
-                target: "runtime::subspace::executor",
+                target: LOG_TARGET,
```

Comment:
For logs it is honestly better to inline strings, it is easier to find usages that way, otherwise you need to do 2 searches